.eslintrc.js

```diff
@@ -297,6 +297,38 @@ module.exports = {
       rules: {
         'no-restricted-syntax': 'warn',
         'no-underscore-dangle': 'off',
+
+        // transloadit rules we would like to enforce in the future
```

Comment:
what does this have to do with jsdoc?
shouldn't this be done in the parent .eslintrc instead?

Reply:
What parent `.eslintrc`? That's the root one. My plan is to harden lint rules for the whole repo (https://github.com/transloadit/uppy/pull/3755), but we need to keep Companion's out of it because there's still too many errors and warnings in its source. I can make it part of another PR, or rename this PR, as you prefer.